### PR TITLE
PUB-2359 Merge channel-management into data-management

### PR DESCRIPTION
**Pull request by KianKwa**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/PUB-2359

### Change description ###

Merge channel-management into data-management

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/pip/data/management/controllers/PublicationController.java [hunk 1]
```diff
@@ -207,7 +207,7 @@ public ResponseEntity<Artefact> uploadPublication(
 
         // Process the created artefact by requesting channel management to generate PDF/Excel files
```
Comment (ChrisS1512) [inline src/main/java/uk/gov/hmcts/reform/pip/data/management/controllers/PublicationController.java:208]: The java doc here needs updating

**Review by ChrisS1512 — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/reform/pip/data/management/controllers/PublicationManagementController.java [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+import uk.gov.hmcts.reform.pip.data.management.models.PublicationFileSizes;
+import uk.gov.hmcts.reform.pip.data.management.service.PublicationManagementService;
+import uk.gov.hmcts.reform.pip.model.authentication.roles.IsAdmin;
+import uk.gov.hmcts.reform.pip.model.publication.FileType;
+
+import java.util.UUID;
+
+@Slf4j
+@RestController
+@Tag(name = "Controller to handle the retrieval of publication files and file' properties, and generation of "
```
Comment (ChrisS1512) [inline src/main/java/uk/gov/hmcts/reform/pip/data/management/controllers/PublicationManagementController.java:25]: I'm wondering about the naming on this file vs Publication Controller. It's hard on first glance to know the split between them.

I wonder if this should be renamed to be something more closely aligned with the extra file formats it gives?

**Review by ChrisS1512 — CHANGES_REQUESTED** [review]

File: src/main/java/uk/gov/hmcts/reform/pip/data/management/controllers/PublicationManagementController.java (hunk 2 above)
Comment (KianKwa) [reply src/main/java/uk/gov/hmcts/reform/pip/data/management/controllers/PublicationManagementController.java]: Updated the description within the tag annotation

File: src/main/java/uk/gov/hmcts/reform/pip/data/management/controllers/PublicationController.java (hunk 1 above)
Comment (KianKwa) [reply src/main/java/uk/gov/hmcts/reform/pip/data/management/controllers/PublicationController.java]: Updated

**Review by ChrisS1512 — APPROVED** [review]

**Review by ChrisS1512 — APPROVED** [review]

**Pull request merged**